CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
--
+- Modify API resource collector to detect if fetched resource is yaml string and
```

Comment:
My only complaint about this patch is that this should not be an enhancement but either a fix or even an internal change.

Reply:
That makes sense, let me do that

Reply:
@Vincent056 if you move the changelog line, I'll lgtm the PR :-)